### PR TITLE
BACKLOG-11104 manage mandatory multiple fields

### PR DESCRIPTION
**Pull request by Taybou**

### CODE REVIEW

File: src/javascript/EditPanelContainer/EditPanel/EditPanelContent/FormBuilder/Section/FieldSet/Field/SelectorTypes/ChoiceList/MultipleSelect/MultipleSelect.jsx [hunk 1]
```diff
@@ -9,37 +9,31 @@ const MultipleSelect = ({field, id, setActionContext}) => {
...
-                        {...formikField}
                         id={id}
                         options={options}
-                        value={
-                            formikField.value &&
-                            options.filter(data => formikField.value.includes(data.value))
-                        }
+                        value={value && options.filter(data => value.includes(data.value))}
                         readOnly={field.readOnly}
-                        onBlur={() => {/* Do Nothing on blur BACKLOG-10095 */}}
```
Comment (FBerthelot) [inline src/javascript/EditPanelContainer/EditPanel/EditPanelContent/FormBuilder/Section/FieldSet/Field/SelectorTypes/ChoiceList/MultipleSelect/MultipleSelect.jsx]: No issue with back button on this one ? :o

File: src/javascript/EditPanelContainer/validation/required.spec.js [hunk 2]
```diff
@@ -95,5 +95,26 @@ describe('required validation', () => {
                 field4: undefined
             });
         });
+
+        it('should return object with all field with errors when all fields are multiple and required', () => {
```
Comment (FBerthelot) [inline src/javascript/EditPanelContainer/validation/required.spec.js]: The test about multiple and boolean field is missing.

File: src/javascript/EditPanelContainer/EditPanel/EditPanelContent/FormBuilder/Section/FieldSet/Field/SelectorTypes/ChoiceList/MultipleSelect/MultipleSelect.jsx (hunk 1 above)
Comment (Taybou) [reply src/javascript/EditPanelContainer/EditPanel/EditPanelContent/FormBuilder/Section/FieldSet/Field/SelectorTypes/ChoiceList/MultipleSelect/MultipleSelect.jsx]: no issue

File: src/javascript/EditPanelContainer/validation/required.spec.js (hunk 2 above)
Comment (Taybou) [reply src/javascript/EditPanelContainer/validation/required.spec.js]: multiple field, boolean or not boolean type it's the same behaviour.

Comment (FBerthelot) [reply src/javascript/EditPanelContainer/validation/required.spec.js]: Same behaviour ? Why adding code for same beahviour ?

I still think one test is missing here

Comment (Taybou) [reply src/javascript/EditPanelContainer/validation/required.spec.js]: where did you see diff code for multiple ?

If you think/see missing, feel free to add :D

**Pull request merged**